src/java.desktop/macosx/native/libawt_lwawt/awt/AWTWindow.m

```diff
@@ -1024,7 +1024,11 @@ - (void)sendEvent:(NSEvent *)event {
             NSRect contentRect = [self.nsWindow contentRectForFrameRect:frame];
 
             // Check if the click happened in the non-client area (title bar)
-            if (p.y >= (frame.origin.y + contentRect.size.height)) {
+            // Also, non-client area includes the edges at left, right and botton of frame
+            if ((p.y >= (frame.origin.y + contentRect.size.height)) ||
+                (p.x >= (frame.origin.x + contentRect.size.width)) ||
+                (fabs(frame.origin.x - p.x) < 4) ||
+                (fabs(frame.origin.y - p.y) < 4)) {
```

Comment:
```suggestion
                (fabs(frame.origin.x - p.x) < 3) ||
                (fabs(frame.origin.y - p.y) < 3)) {
```
It is not critical here, but it calls `deliverNCMouseDown` even if the resize cursor is not shown for the left and bottom window borders.